lib/ansible/module_utils/network/nxos/cmdref/telemetry/telemetry.py

```diff
@@ -0,0 +1,145 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Cisco and/or its affiliates.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Telemetry Command Reference File
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+TMS_GLOBAL = '''
+# The cmd_ref is a yaml formatted list of module commands.
+# A leading underscore denotes a non-command variable; e.g. _template.
+# TMS does not have convenient global json data so this cmd_ref uses raw cli configs.
+---
+_template: # _template holds common settings for all commands
+  # Enable feature telemetry if disabled
+  feature: telemetry
+  # Common get syntax for TMS commands
+  get_command: show run telemetry all
+  # Parent configuration for TMS commands
+  context:
+    - telemetry
+certificate:
+  _exclude: ['N3K', 'N5K', 'N6k', 'N7k']
```

Comment:
I think these `_exclude` keys should be in `_template` instead of each command key.

Reply:
Do we support that in CmdRef today?

Reply:
Didn't need it before. We should have it though. 

This might do it:

```
        # Remove excluded commands (no platform support for command)
+       if plat in ref[_template].get('_exclude', ''):
+           ref['commands'] = []
+           # raise/warn  ?
+           return

        for k in ref['commands']:
            if plat in ref[k].get('_exclude', ''):
                ref['commands'].remove(k)
```

Ref:
https://github.com/ansible/ansible/blob/c963dae27427cc695c5c9419d7a8c26914ad8939/lib/ansible/module_utils/network/nxos/nxos.py#L836-L839